airbrakes/data_handling/packets/logger_data_packet.py

```diff
@@ -1,16 +1,16 @@
 """Module for describing the data packet for the logger to log"""
 
-from typing import Required, TypedDict
+from typing import TypedDict
 
 
-class LoggedDataPacket(TypedDict, total=False):  # total=False means all fields are NotRequired
+class LoggerDataPacket(TypedDict, total=False):  # total=False means all fields are NotRequired
     """
     Represents a collection of all data that the logger can log in a line. Not every field will be
     filled in every packet.
     """
 
-    state: Required[str]
-    extension: Required[str]
+    state: str
+    extension: str
```

Comment:
need to add the other ContextPacket fields & add a comment saying that it is of a ContextPacket